src/braket/default_simulator/state_vector_simulator.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import sys
+
+from braket.default_simulator.simulator import BaseLocalSimulator
+from braket.default_simulator.state_vector_simulation import StateVectorSimulation
+from braket.device_schema.simulators import (
+    GateModelSimulatorDeviceCapabilities,
+    GateModelSimulatorDeviceParameters,
+)
+from braket.ir.jaqcd import Program
+from braket.task_result import GateModelTaskResult
+
+
+class DefaultSimulator(BaseLocalSimulator):
+    def run(
+        self, circuit_ir: Program, qubit_count: int, shots: int = 0, batch_size: int = 1,
```

Comment:
This appears to change the existing behavior. `batch_size` could only be passed in as a kwarg earlier I think.